docs/en/cli.md

```diff
@@ -0,0 +1,475 @@
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
+* des\_endpoint The destination node for migration
+
+```
+> migrate 172.27.2.52:9991 table1 1 172.27.2.52:9992
+partition migrate ok
+> migrate 172.27.2.52:9991 table1 1-5 172.27.2.52:9992
+partition migrate ok
+> migrate 172.27.2.52:9991 table1 1,2,3 172.27.2.52:9992
+partition migrate ok
+```
+
+#### confget
+
+Get configuration information, currently only supports auto\_failover
+
+Command format: `confget [conf_name]`
+
+* conf\_name configuration item name, optional
+
+```
+> confget
+  key value
+-----------------------------
+  auto_failover false
+> confget auto_failover
+  key value
+------------------------
+  auto_failover false
+```
+
+#### confsets
+
+Modify configuration information, currently only supports auto\_failover
+
+Command format: `confset conf_name value`
+
+* conf\_name configuration item name
+* value The value set by the configuration item
+
+```
+> confset auto_failover true
+set auto_failover ok
+```
+
+#### offlineendpoint
+
+Offline node. This command is asynchronous and after the successful return, you can view the running status through showopstatus
+
+Command format: `offlineendpoint endpoint [concurrency]`
+
+* endpoint is the endpoint of the failed node. This command will perform the following operations on all shards under the node:
+  * If it is the master, execute the re-election of the master
+  * If it is a slave, find the master node and delete the current endpoint copy from the master node
+  * Modify is_alive status to no
+* concurrency controls the concurrent number of task execution. This configuration is optional, the default is 2 (name_server_task_concurrency configuration can be configured), and the maximum value is the value configured by name_server_task_max_concurrency
+
+```bash
+> offlineendpoint 172.27.128.32:8541
+offline endpoint ok
+>showtable
+  name tid pid endpoint role ttl is_alive compress_type offset record_cnt memused
+-------------------------------------------------- -------------------------------------------------- ------------------
+  flow 4 0 172.27.128.32:8541 leader 0min no kNoCompress 0 0 0.000
+  flow 4 0 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  flow 4 0 172.27.128.31:8541 follower 0min yes kNoCompress 0 0 0.000
+  flow 4 1 172.27.128.33:8541 leader 0min yes kNoCompress 0 0 0.000
+  flow 4 1 172.27.128.31:8541 follower 0min yes kNoCompress 0 0 0.000
+  flow 4 1 172.27.128.32:8541 follower 0min no kNoCompress 0 0 0.000
+```
+
+After the command is executed successfully, all shards will have leaders in the yes state
+
+#### recoverendpoint
+
+Restore node data. This command is asynchronous and after the successful return, you can view the running status through showopstatus
+
+Command format: `recoverendpoint endpoint [need_restore] [concurrency]`
+
+* endpoint is the endpoint of the node to restore
```

Comment:
* endpoint: the endpoint of the node to restore

Reply:
Ok, thank you.